modules/notification/action/action.go

```diff
@@ -92,13 +92,22 @@ func (a *actionNotifier) NotifyCreateIssueComment(doer *models.User, repo *model
 	act := &models.Action{
 		ActUserID: doer.ID,
 		ActUser:   doer,
-		Content:   fmt.Sprintf("%d|%s", issue.Index, comment.Content),
 		RepoID:    issue.Repo.ID,
 		Repo:      issue.Repo,
 		Comment:   comment,
 		CommentID: comment.ID,
 		IsPrivate: issue.Repo.IsPrivate,
 	}
+
+	content := ""
+
+	if len(comment.Content) > 200 {
+		content = comment.Content[0:200] + " ..."
```

Comment:
```suggestion
		content = comment.Content[0:200] + "…"
```

Reply:
```suggestion
		content = content[:strings.LastIndex(comment.Content[0:200], " ")] + "…" 
```

would make it break on the last full word of the 200 char limit and might look a bit nicer. Not sure in code is actually the right place to do this though wouldn't this also truncate the text in email as well which seems unnecessary? Shouldn't this be done in template?

Reply:
If Im correct it's only for action's and if we would store string never shown to the user into db witch exist elsewhere isnt this just waste of DB space?

Reply:
It won't impact other notify ways, because they will get data from issues directly. not from action. 
ref:

https://github.com/go-gitea/gitea/blob/06480af4723d34ac744c5f4f52fd974c3d016e59/modules/notification/mail/mail.go#L29-L47